src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -417,7 +417,7 @@ export default class OrganizationDiscover extends React.Component {
 
         <DiscoverGlobalSelectionHeader
           organization={organization}
-          project={currentQuery.projects}
+          projects={currentQuery.projects}
```

Comment:
This isn't listed in any proptypes anywhere and the styled component doesn't use it either. afaik it's just a good old fashioned typo.

Reply:
bit it does cause an invalid proptype error

Reply:
Hmm can we revert this change just in case?

Reply:
unfortunately skipping this breaks tests because `withProjects` takes a proptype called `project`. 

We can check with @lynnagara to see if anything is missing here, but I went through all of the code in `GlobalSelectionHeader` as well as the wrappers and didn't see anything that looked for `project`. 

This is also the only instance of `GlobalSelectionHeader` that takes any kind of `projects` property. Every other instance passes only `organization`, except for monitors which passes some extra flags to hide stuff. While `projects` was at one point a prop, I don't think we have used it in a meaningful way in some time.

I also did some tests against master from before the weekend and everything regarding the discover view seems to be the same, including saved searches which don't alter that state of the header.

Reply:
I'd guess it's a typo, might be good to do a bit of manual testing on this though

Reply:
I have done fabulous amounts of manual testing, and as long as saved queries aren't supposed to update the header then everything looks right.

Reply:
I also tested sharing projects from user to user and it looks 👌👌👌

Reply:
Sounds good, thanks for double checking